playwright/integrations/unauthenticated-chat.spec.ts

```diff
@@ -541,6 +542,40 @@ test.describe('UnauthenticatedChat @UnauthenticatedChat', () => {
         expect(sessionCloseResponse.status()).toBe(200);
     });
 
+    test('ChatSDK.endChat() should not call close session for identified sessions ended by agent', async ({ page }) => {

```

Comment:
`ChatSDK.endChat() with optional params \'isSessionEnded\' set as \'true\' should call not sessionClose`